runtime/pubsub/types.go

```diff
@@ -1,38 +1,17 @@
 package pubsub
 
 import (
-	"context"
-
-	internal "encore.dev/pubsub/internal/types"
+	"encore.dev/pubsub/internal/types"
 )
 
-// SubscriptionConfig is used when creating a subscription
```

Comment:
Maybe would be nicer if `publicapigen` uses this documentation instead? I could imagine we want the `internal/types` package to contain rather internal documentation than external-facing documentation.

Reply:
I wanted them copied from the internal docs, because we'd keep the docs more up to date in the place the thing is actually defined.

That said - I've just made it so we only copy the doc's over if doc's are missing from the target location (i.e. we've not written explicit docs in the public location)